frontend_tests/node_tests/unread.js

```diff
@@ -5,7 +5,18 @@ zrequire('util');
 zrequire('unread');
 
 set_global('blueslip', {});
-set_global('page_params', {});
+set_global('settings_notifications', {
+    desktop_icon_count_display_values: {
+        messages: {
+            code: 1,
+            description: "All unreads",
+        },
+        notifiable: {
+            code: 2,
+            description: "Private messages and mentions",
+        },
+    },
+});
```

Comment:
This would be better done by just importing `settings_notifications`.  In general, we never want to copy data like this from the production code for unit tests, because it risks being out of date.